src/aggregation/mod.rs

```diff
@@ -239,6 +248,10 @@ pub enum Key {
     F64(f64),
 }
 
+trait MergeFruits {
```

Comment:
Is this trait useful? I'm not sure I understand its point.

Reply:
Ah it is used to share code in the merge_maps thingy?

Why is it not declared in `intermediate_agg_result` then, close to that function then?

Reply:
yes, so multiple buckets can share the merge code on collections

I moved it to intermediate_agg_result